device/cuda/src/seeding/track_params_estimation.cu

```diff
@@ -69,7 +82,7 @@ track_params_estimation::operator()(
     unsigned int num_blocks = seeds_size / num_threads + 1;
```

Comment:
You should fix this calculation. This is not the right way of calculating the number of blocks.